### ISSUE TITLE
predictSmooth vs plotSmoothers

### ISSUE THREAD

**Issue by flde** [statOmics/tradeSeq#230]
Hello tradeSeq-Team, 

I plotted the gene expression with plotSmoothers and my own function where I use predictSmooth. However, for some genes the results are similar (e.g. Kit) but for others (e.g. Cd47) the smoothed signal looks quite different. 

The only thing I noticed is that both function use .getPredictRangeDf() and predictGAM(). But plotSmoothers then removes (?) the offset `yhat <- c(exp(t(Xdf %*% t(beta)) + df$offset)` while that happens later with predictSmooth? 

I would highly appreciate it if you can help me to understand why the smoothed lines differ between the two functions. 

![smoother](https://user-images.githubusercontent.com/20926246/227952237-dcaf512b-40b4-4634-adfb-25f9481b1754.png)

Best wishes, 
Florian

**Comment by koenvandenberge** [comment]
Hi @flde,

You may want to share your own function to allow us to look into this or take a look at `plotSmoothers` yourself.

Also it looks like the points can have here have different on your two plots, did you set the same seed prior to running `fitGAM`? It may just be a plotting thing.

**Comment by flde** [comment]
Hello @koenvandenberge, 

Many thanks for your help! I did a check with your data from the workshop. The problem was that I did not take the log1p from the output of predictSmooth. I attached the  jupyter notebook in case someone else has the same question. 

[smoother.zip](https://github.com/statOmics/tradeSeq/files/11097844/smoother.zip)

Best, 
Florian 

![download](https://user-images.githubusercontent.com/20926246/228463906-8d02a3cf-9fd5-48f5-8a12-8b0fbf90603f.png)
